src/sync/Synchroniser.js

```diff
@@ -133,14 +177,15 @@ export class Synchroniser {
     const thisSiteId = this.settings.get(SYNC_SITE_ID);
     const serverId = this.settings.get(SYNC_SERVER_ID);
     const response = await fetch(
-      `${serverURL}/sync/v2/queued_records/?from_site=${thisSiteId}&to_site=${serverId}`,
+      `${serverURL}/sync/v2/queued_records?from_site=${thisSiteId}&to_site=${serverId}`,
       {
         method: 'POST',
         headers: {
           Authorization: this.authenticator.getAuthHeader(),
         },
         body: JSON.stringify(records),
-      });
+      }
```

Comment:
I know it's 'prettier' doing this, but doesn't it break the eslint rule about dangling commas?